cross/psutil-3.2.1/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME = psutil
+PKG_VERS = 3.2.1
```

Comment:
Cool, I'll check if OctoPrint works with 3.3.0 when the new Python packages are released.